aiden-recommender/aiden_recommender/scrapers/field_extractors.py

```diff
@@ -0,0 +1,127 @@
+from pydantic import BaseModel
+from abc import ABC, abstractmethod
+import jmespath
+from datetime import datetime
+from pydantic_core import ValidationError
+from loguru import logger
+
+
+class AbstractFieldExtractor(ABC):
+    @staticmethod
+    def select(data: dict, query: str) -> any:
```

Comment:
Will it ever be used separately ? If not shouldn't it be part of the extract method ?